src/modules/userInterface.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] There is no unit test written in your project. Please add unit tests for the counter feature of the app using [the Jest framework](https://jestjs.io/).